legal-api/report-templates/dissolution.html

```diff
@@ -34,7 +34,6 @@
       <div class="container">
          {% if business.legalType in ['GP', 'SP'] %}
             [[dissolution/firmsDissolutionDate.html]]
-            [[common/completingParty.html]]
```

Comment:
Should this template part be deleted also?

Reply:
yeah! i don't see any other reference for this file as well